Libraries/LibJS/Lexer.h

```diff
@@ -37,14 +38,9 @@ namespace JS {
 class Lexer {
 public:
     explicit Lexer(StringView source);
-    Lexer(StringView source, bool log_errors)
-        : Lexer(source)
-    {
-        m_log_errors = log_errors;
-    }
+    // explicit Lexer(StringView source, AK::Function<void(String message, size_t line, size_t column)> on_error);
```

Comment:
With the new error-free lexer, this won't be needed anymore and should probably be removed. I guess that it was leftover from the lexer refactoring ?

Reply:
Yep, that was me playing around with a lexer-just-reports-error-someone-else-takes-care-of-it approach. Thanks!